a10_nlbaas2oct/driver.py

```diff
@@ -375,7 +375,15 @@ def main():
         conf_lb_id_list = []
 
     if CONF.lb_id:
+        if not db_utils.get_loadbalancer_entry(n_session, CONF.lb_id):
+            print('Error: Provide valid --lb-id value.')
+            return
         conf_lb_id_list.append(CONF.lb_id)
+
+    if CONF.project_id and db_sessions.get('k_session'):
```

Comment:
The keystone db connection is optional in configuration. If we want this extra layer of checking then we need to make it required.